cli/daemon/daemon.go

```diff
@@ -0,0 +1,74 @@
+package daemon
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/ttlog"
+)
+
+// DaemonCtx contains information for running an daemon instance.
+type DaemonCtx struct {
+	// Port is a port number to be used for daemon http server.
+	Port int
+	// PIDFile is a path of a file contains pid of daemon process.
+	PIDFile string
+	// LogPath is a path to a file contains log of daemon process.
+	LogPath string
+	// LogMaxSize is the maximum size in megabytes of the log file
+	// before it gets rotated. It defaults to 100 megabytes.
+	LogMaxSize int
+	// LogMaxBackups is the maximum number of old log files to retain.
+	// The default is to retain all old log files (though LogMaxAge may
+	// still cause them to get deleted).
+	LogMaxBackups int
+	// LogMaxAge is the maximum number of days to retain old log files
+	// based on the timestamp encoded in their filename. Note that a
+	// day is defined as 24 hours and may not exactly correspond to
+	// calendar days due to daylight savings, leap seconds, etc. The
+	// default is not to remove old log files based on age.
+	LogMaxAge int
+	// ListenInterface is a network interface the IP address
+	// should be found on to bind http server socket.
+	ListenInterface string
+	// Control UNIX socket for started instance.
+	ConsoleSocket string
```

Comment:
What for the daemon most have a Console socket?

Reply:
Removed `ConsoleSocket` field.

Reply:
![image](https://user-images.githubusercontent.com/57662928/201894805-b3e9c228-8e88-4d7f-8d5e-82ae6a068ea8.png)
It looks like the field has not been deleted)